plugin/src/main/java/org/screamingsandals/bedwars/config/Configurator.java

```diff
@@ -496,6 +496,8 @@ public void createFiles() {
         checkOrSetConfig(modify, "turnOnExperimentalGroovyShop", false);
         checkOrSetConfig(modify, "preventSpectatorFlyingAway", false);
         checkOrSetConfig(modify, "removePurchaseMessages", false);
+        checkOrSetConfig(modify, "removePurchaseFailedMessages", false);
```

Comment:
Because removePurchaseMessages was used before for failures as well, it should automatically migrate this value.

Reply:
ok wait

Reply:
latest commit probably resolves this

Reply:
use getBoolean instead of get

Reply:
changed